crates/telemetry/src/env.rs

```diff
@@ -8,6 +8,7 @@ use opentelemetry_otlp::{
 const OTEL_SDK_DISABLED: &str = "OTEL_SDK_DISABLED";
 const OTEL_EXPORTER_OTLP_TRACES_PROTOCOL: &str = "OTEL_EXPORTER_OTLP_TRACES_PROTOCOL";
 const OTEL_EXPORTER_OTLP_METRICS_PROTOCOL: &str = "OTEL_EXPORTER_OTLP_METRICS_PROTOCOL";
+const APP_LOG_TO_TRACING_EVENT_DISABLED: &str = "APP_LOG_TO_TRACING_EVENT_DISABLED";
```

Comment:
Is this name forced on us by OTel or do we have leeway to change it?  (I ask because if it's Spin-specific then we might want to give it a `SPIN_` prefix.)

Reply:
Purely from the land of my own creativity 😄. `SPIN_` prefix is a great point. Also open to workshopping the name. I struggled to achieve brevity and clarity on this complex of a concept.

Reply:
`SPIN_DISABLE_LOG_TO_TRACING`?